context/app/static/js/components/entity-tile/EntityTileBottom/EntityTileBottom.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import format from 'date-fns/format';
+
+import { FixedWidthFlex, StyledDivider, StyledTypography, StyledDatasetIcon, StyledSampleIcon } from './style';
+
+function EntityTileBottom(props) {
+  const { isCurrentEntity, entityData, descendantCounts } = props;
+
+  return (
+    <FixedWidthFlex $isCurrentEntity={isCurrentEntity}>
+      {Object.entries(descendantCounts).map(([k, v]) => (
+        <>
+          {k === 'Dataset' ? <StyledDatasetIcon /> : <StyledSampleIcon />}
```

Comment:
```suggestion
          {k === 'Dataset' && <StyledDatasetIcon />}
          {k === 'Sample' && <StyledSampleIcon />}
```
Slightly paranoid, probably.